doc/classes/ParticlesMaterial.xml

```diff
@@ -266,6 +281,36 @@
 		<member name="tangential_accel_min" type="float" setter="set_param_min" getter="get_param_min" default="0.0">
 			Minimum tangential acceleration.
 		</member>
+		<member name="turbulence_active" type="bool" setter="set_turbulence_active" getter="get_turbulence_active" default="false">
+			Enables and disables Turbulence for the particle system.
+		</member>
+		<member name="turbulence_influence_max" type="float" setter="set_param_max" getter="get_param_max">
+			Minimum turbulence influence on each particle.
+		</member>
+		<member name="turbulence_influence_min" type="float" setter="set_param_min" getter="get_param_min">
+			Maximum turbulence influence on each particle.
+		</member>
+		<member name="turbulence_influence_over_life" type="Texture2D" setter="set_param_texture" getter="get_param_texture">
+			Each particle's amount of turbulence will be influenced along this [CurveTexture] over its life time.
+		</member>
+		<member name="turbulence_initial_displacement_max" type="float" setter="set_param_max" getter="get_param_max">
+			Maximum displacement of each particles spawn position by the turbulence.
+		</member>
+		<member name="turbulence_initial_displacement_min" type="float" setter="set_param_min" getter="get_param_min">
+			Minimum displacement of each particles spawn position by the turbulence.
+		</member>
+		<member name="turbulence_noise_scale" type="float" setter="set_param_direct" getter="get_param_direct">
+			This value controls the overall scale of the turbulence noise pattern.
+		</member>
+		<member name="turbulence_noise_speed" type="Vector3" setter="set_turbulence_noise_speed" getter="get_turbulence_noise_speed">
+			The movement speed of the turbulence pattern.
+		</member>
+		<member name="turbulence_noise_speed_random" type="float" setter="set_param_direct" getter="get_param_direct">
```

Comment:
Interesting thought... Although it would mean another additional parameter.
I will try it today and see how it looks. If it's a big and worty difference, I need to reconsider this!

Reply:
Ok, I tested it and using a min/max value on the randomness led to "undesireable" results.
What happened is that it "diffused/blurred" the noise pattern so much that even a min/max of 0.98/1.0 is enough to bring so much chaos into the noise that it's more or less white noise compared to a fixed value of 0.99 (the median of this example).

So while I like the idea, it's not working in a practical application 😐